src/DIRAC/Resources/Computing/BatchSystems/Condor.py

```diff
@@ -106,15 +107,15 @@ def submitJob(self, **kwargs):
     Output = $(Cluster).$(Process).out
     Error = $(Cluster).$(Process).err
     Log = test.log
-    Environment = CONDOR_JOBID=$(Cluster).$(Process)
+    Environment = "CONDOR_JOBID=$(Cluster).$(Process) DIRAC_PILOT_STAMP=$(stamp)"
```

Comment:
```suggestion
    Environment = "CONDOR_JOBID=$(Cluster).$(Process) DIRAC_PILOT_STAMP=$(stamps)"
```

Reply:
I do not think this is correct. This is similar to the HTCondorCE case, the stamp is defined in the Queue command below